http-mclient/src/Network/HTTP/MClient.hs

```diff
@@ -120,3 +123,18 @@ getContentType response
   . maybe (pure $ mediaType @'PlainText) Media.parseAccept
   . List.lookup hContentType
   $ HTTP.responseHeaders response
+
+addMediaHeaders
+  :: forall ctyp . HasMediaType ctyp
+  => HTTP.Request
+  -> HTTP.Request
+addMediaHeaders request = request
+  { HTTP.requestHeaders
+      = [ (hContentType, headerValue)
+        , (hAccept, headerValue)
+        ]
+        <> HTTP.requestHeaders request
+  }
+  where
+    mediaType' = mediaType @ctyp
+    headerValue = CI.original $ Media.mainType mediaType' <> "/" <> Media.subType mediaType'
```

Comment:
@epicallan I'd probably have the code aligned at `=`.

Reply:
@snusnu Addressed, thanks.